csuntechlab/iSTART
# STR-160 -- FE: Update informational template to allow for different image placement

# Description
Updated info template with new JSON template.
  
# Testing
1. Login as `cyndee.bellamy` and `april.feldman`.
2. Visit dashboard and click the `alcohol` module.
3. Visit all informational slides and ensure it meets the alcohol power point image positioning.
